app/src/main/java/de/westnordost/streetcomplete/quests/bollard_type/BollardType.kt

```diff
@@ -1,10 +1,13 @@
 package de.westnordost.streetcomplete.quests.bollard_type
 
-enum class BollardType(val osmValue: String) {
+sealed interface BollardTypeAnswer
+
+enum class BollardType(val osmValue: String) : BollardTypeAnswer {
     RISING("rising"),
     REMOVABLE("removable"),
     FOLDABLE("foldable"),
     FLEXIBLE("flexible"),
     FIXED("fixed"),
-    NOT_BOLLARD("not_bollard"),
 }
+
+object DifferentBarrierType : BollardTypeAnswer
```

Comment:
IMO this should have something like `NotBollardBut...` or similar in its name. After all, this is a public singleton, not an enum constant. Enum constants are accessed like BollardType.FOLDABLE. Public singletons are directly accessed. Hence, we should avoid using the exact same names for two different singletons (i.e. there is also `object DifferentBarrierType : BicycleBarrierTypeAnswer`)